credhub.yml

```diff
@@ -0,0 +1,114 @@
+# CredHub release file location must be provided 
+- type: replace
+  path: /releases/-
+  value:
+    name: credhub
+    url: ((file_path_to_credhub_release))
+
+# A chosen dev_key value must be provided in the form of 32 hex characters 
+- type: replace
+  path: /instance_groups/name=bosh/jobs/-
+  value:
+    name: credhub
+    release: credhub
+    properties:
+      credhub: 
+        user_management:
+          uaa:
+            url: "https://((internal_ip)):8443"
+            verification_key: ((uaa_jwt_signing_key.public_key))
+        data_storage:
+          type: postgres
+          host: 127.0.0.1
+          port: 5432
+          username: postgres
+          password: ((postgres_password))
+          database: credhub
+        tls: ((credhub_tls))
+        encryption:
+          provider: dev_internal
+          dev_key: ((credhub_dev_key))
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/postgres/additional_databases?/-
+  value: credhub
+
+# Configure Director
+- type: replace
+  path: /instance_groups/name=bosh/properties/director/config_server?
+  value:
+    enabled: true
+    url: "https://((internal_ip)):8844/api/"
+    ca_cert: ((credhub_tls.ca))
+    uaa:
+      url: "https://((internal_ip)):8443"
+      ca_cert: ((uaa_ssl.ca))
+      client_id: director_to_credhub
+      client_secret: ((uaa-clients-director-to-credhub))
+
+# Configure UAA
+- type: replace
+  path: /instance_groups/name=bosh/jobs/name=uaa/properties/uaa/clients/director_to_credhub?
+  value:
+    override: true
+    authorized-grant-types: client_credentials
+    scope: ""
+    authorities: credhub.read,credhub.write
+    access-token-validity: 3600
+    secret: ((uaa_clients_director_to_credhub))
+
+- type: replace
+  path: /instance_groups/name=bosh/jobs/name=uaa/properties/uaa/clients/credhub?
+  value:
+    override: true
+    authorized-grant-types: password,refresh_token
+    scope: credhub.read,credhub.write
+    authorities: ""
+    access-token-validity: 60
+    refresh-token-validity: 1800
+    secret: ""
+
+- type: replace
+  path: /instance_groups/name=bosh/jobs/name=uaa/properties/uaa/scim/users?/-
```

Comment:
This is a seeded user to access the CredHub CLI.

Now that I am thinking of it, perhaps it is better to just add the appropriate scope to the existing admin user. I'm not sure our perspective on whether the seeded admin user should be only a BOSH admin or an admin of everything. Thoughts?